app/src/main/java/org/mozilla/fenix/utils/Settings.kt

```diff
@@ -590,9 +590,8 @@ class Settings(private val appContext: Context) : PreferencesHolder {
         true
     )
 
-    val enabledTotalCookieProtection: Boolean by lazy {
-        FxNimbus.features.engineSettings.value().totalCookieProtectionEnabled
-    }
+    val enabledTotalCookieProtection: Boolean
+        get() = FxNimbus.features.engineSettings.value().totalCookieProtectionEnabled
```

Comment:
Are there any other uses of this lazy delegate pattern overcaching nimbus values in the code?